app/components/windows/NameScene.vue.ts

```diff
@@ -25,28 +26,47 @@ export default class NameScene extends Vue {
   @Inject()
   windowsService: WindowsService;
 
-  options: { sceneToDuplicate?: string, rename?: string } = this.windowsService.getChildWindowQueryParams();
+  @Inject()
+  selectionService: ISelectionServiceApi;
+
+  options: {
+    sceneToDuplicate?: string,
+    rename?: string,
+    itemsToGroup?: string[]
+  } = this.windowsService.getChildWindowQueryParams();
 
   mounted() {
     this.name = this.options.rename ?
       this.options.rename :
-      this.sourcesService.suggestName(this.options.sceneToDuplicate || 'NewScene');
+      this.sourcesService.suggestName(
+        this.options.sceneToDuplicate ||
+        this.options.itemsToGroup ? 'Group' : 'NewScene'
```

Comment:
Just a minor suggestion, but it might be cool to name this dynamically based on the name of one of the scene items they are grouping.  For example, the pseudocode might look like:
```
const firstSceneItemName = sceneItemsToGroup[0].name;
const name = this.suggestName(`${firstSceneItemName} Group`);
```

Reply:
Also for the new scene case it should be "New Scene" (with a space in between)